doc/source/whatsnew/v0.19.0.txt

```diff
@@ -894,6 +894,7 @@ Removal of prior version deprecations/changes
 - ``Panel.shift()`` has dropped the ``lags`` parameter in favour of ``periods`` (:issue:`14041`)
 - ``pd.Index`` has dropped the ``diff`` method in favour of ``difference`` (:issue:`13669`)
 
+- ``pd.DataFrame`` has dropped the ``to_wide`` method in favour of ``to_panel`` (:issue:`14039`)
```

Comment:
can you audito the ``favour`` -> ``favor`` (or just pick one) for all of these lines. I know trivial.......